### PR TITLE
Add paddle.nn.functional.loss NN Loss Functions to Paddle Frontend #15053

### LINKED ISSUE #21312: nll_loss

### PR DESCRIPTION
**Pull request by aks1204**
Close #21312

### CODE REVIEW

**Comment by aks1204** [comment]
- [ ] #15053

**Comment by aks1204** [comment]
- [ ] #21312

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by aks1204** [comment]
@sagewhocodes could you review my pull request

**Comment by sagewhocodes** [comment]
Hello @aks1204 , 
Please make the required changes and add the test for the frontend function, please refer [Ivy Frontend Tests](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html) for more information.

Thank your for contributing to Ivy 😄

**Comment by aks1204** [comment]
@sagewhocodes I've made the necessary changes. Could you please review my pull request.

**Comment by aks1204** [comment]
@sagewhocodes could you please review my pull request. I have added the frontend tests. My PR is however still failing some checks. I don't know if it has to do anything with my code. Could you please help and look into it? Thanks.

**Comment by aks1204** [comment]
@sagewhocodes also if you won't mind can you share your discord handle. Thanks.

**Comment by aks1204** [comment]
@sagewhocodes I've checked for my code once again. I think I've implemented my function correctly. I don't think the checks my PR is failing is due to my code. Could you tell why my PR is failing some checks and if the issue is specific to my code(and if so what part of my code).

**Comment by aks1204** [comment]
@sagewhocodes I've resolved some conflicts. They weren't their when I created my PR.
